ray-operator/apis/ray/v1alpha1/rayservice_types.go

```diff
@@ -1,22 +1,85 @@
 package v1alpha1
 
 import (
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+type ServiceStatus string
+
+const (
+	FAIL_TO_GET_RAYSERVICE           ServiceStatus = "fail_to_get_rayservice"
+	FAIL_TO_GET_OR_CREATE_RAYCLUSTER ServiceStatus = "fail_to_get_or_create_RAYCLUSTER"
+	WAIT_FOR_DASHBOARD               ServiceStatus = "wait_for_dashboard"
+	FAIL_SERVE_DEPLOY                ServiceStatus = "fail_serve_deploy"
+	FAIL_GET_SERVE_DEPLOYMENT_STATUS ServiceStatus = "fail_get_serve_deployment_status"
+	RUNNING                          ServiceStatus = "running"
+	RESTARTING                       ServiceStatus = "restarting"
+	FAIL_DELETE_RAYCLUSTER           ServiceStatus = "fail_delete_raycluster"
+)
+
+// ServeConfigSpec defines the desired state of RayService
+type ServeConfigSpec struct {
+	Name                      string             `json:"name"`
+	ImportPath                string             `json:"import_path"`
```

Comment:
The json fields should be lowerCamelCase -- importPath, etc.

Reply:
I do not think so. `import_path` is defined by Ray Serve Endpoint.